hathor/indexes/memory_utxo_index.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import defaultdict
+from dataclasses import dataclass, field
+from typing import DefaultDict, Iterator, Optional, Set, Tuple
+
+from sortedcontainers import SortedList
+from structlog import get_logger
+
+from hathor.indexes.utxo_index import UtxoIndex, UtxoIndexItem
+
+logger = get_logger()
+
+
+@dataclass(frozen=True)
+class _IndexItem:
```

Comment:
Use NamedTuples for each of these items.

Reply:
Done.

Reply:
Thanks!